src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetConvictionsForPersonServiceTest.kt

```diff
@@ -38,12 +34,12 @@ internal class GetOffencesForPersonServiceTest(
         Response(data = listOf(Person(firstName = "Chandler", lastName = "Bing", prisonerId = prisonerNumber))),
       )
 
-      whenever(nomisGateway.getOffencesForPerson(prisonerNumber)).thenReturn(
+      whenever(nomisGateway.getConvictionsForPerson(prisonerNumber)).thenReturn(
         Response(
           data = listOf(
-            Offence(code = "RR12345", description = "First Offence", date = LocalDate.parse("2020-02-03")),
-            Offence(code = "RR54321", description = "Second Offence", date = LocalDate.parse("2021-03-04")),
-            Offence(code = "RR24680", description = "Third Offence", date = LocalDate.parse("2022-04-05")),
+            Conviction(code = "RR12345", description = "First Offence", date = LocalDate.parse("2020-02-03")),
```

Comment:
I wonder if the description here should be First/Second/Third conviction rather than offence? thoughts?